src/Extensions/PageHealthExtension.php

```diff
@@ -35,14 +36,17 @@ class PageHealthExtension extends DataExtension
     public function updateCMSFields(FieldList $fields)
     {
         parent::updateCMSFields($fields);
-
-        $fields->addFieldsToTab('Root.Main', [
-            ToggleCompositeField::create(null, 'SEO Health Analysis', [
-                GoogleSearchPreview::create('GoogleSearchPreview', 'Search Preview', $this->getOwner(), $this->getRenderedHtmlDomParser()),
-                TextField::create('FocusKeyword', 'Set focus keyword'),
-                HealthAnalysisField::create('ContentAnalysis', 'Content Analysis', $this->getOwner()),
-            ])
-        ], 'Metadata');
+        
+        $errorPageClass = ErrorPage::class;
+        if(!$this->owner instanceof $errorPageClass){
```

Comment:
Just one minor thing, a great practise to get used to is "return early".

Can this become:

```
if ($this->owner instanceof ErrorPage::class) { return; }
```

That way the rest of the code doesn't need to be indented,

Appreciate this!

Reply:
It will cause an error:
[Emergency] Uncaught ParseError: syntax error, unexpected 'class' (T_CLASS), expecting variable (T_VARIABLE) or '$'
We can use is_a() function, but it's slower than instanceof 
But we can use:
```
if ($this->owner instanceof SilverStripe\ErrorPage\ErrorPage) {
            return;
        }
```

I have made one more request